CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Since v0.10.0 this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Unreleased version
 
```

Comment:
Just a minor nit: I wonder if we should get into the habit of adding placeholder content (e.g. "TBD" or "No changes yet") in the "Unreleased version" section of the changelog, when cutting a new release as we're doing now. Otherwise it may be slightly confusing.

Also, I'm trying to figure out if some other heading for this section would be clearer — perhaps "Unreleased changes", "Upcoming version", or "Next version".

Neither of these changes are significant, so just let me know what you think — I'm happy to keep things as they are if you feel these suggestions bring no improvement.

Reply:
Looks like the current heading had been discussed previously in https://github.com/hamstar/Wikimate/pull/83#discussion_r105555489. Some of that discussion would go against the "Unreleased changes" proposal (since they are already released to the public, even if not included in a tagged version), so that would leave us with "Upcoming version" or "Next version" (or something along those lines).

Reply:
Yes, I just reread that discussion earlier this week too. I agree having the empty "unreleased" header in a newly cut release is a bit confusing, but rather than changing the wording I think the clean solution is to not include it in the tag at all. This looks like a slight deviation from the keepachangelog guidelines, but would easily settle this academic wording discussion.

Then, with the next PR (which will normally also contain a changelog entry), that is the moment to add back the "Unreleased version" header. WDYT?
If you agree, the governance doc needs amendments on this point too.

Reply:
Hmm... that would work, but it would require regular PRs to have more complex instructions regarding the changelog entry (a conditional addition of the header for unreleased changes if it doesn't exist yet). I'm more comfortable having extra steps in the release process, which already requires tweaking the section headers of the CHANGELOG file anyway.

I like having "No changes yet" as the section content, since that makes things explicit and intuitive in all the steps involved (both cutting a new release and making new PRs). But I won't insist further if you push back and prefer your approach.

Reply:
A stub text to give context to the header also works for me. Done in e2c026c

Reply:
Resolving this conversation, since the changes have been moved to #105, which has already been merged.